### PR TITLE
updated docs for reporting

### PR DESCRIPTION
**Pull request by jongyllen**
updated docs for reporting

### CODE REVIEW

File: docs/sources/enterprise/reporting.md [hunk 1]
```diff
@@ -42,6 +42,16 @@ Currently only Organization Admins can create reports.
 
 {{< docs-imagebox img="/img/docs/enterprise/reports_create_new.png" max-width="500px" class="docs-image--no-shadow" >}}
 
+## Send test mail
```
Comment (oddlittlebird) [inline docs/sources/enterprise/reporting.md:45]: Try rewriting this as a numbered steps. The task is Send a test mail.

1. In the (place), click **Send test mail**.
1. In the (field name), enter the email address or addresses that you want to test. (Do you need to separate them by a comma?
  If you want to use email addresses from the report, then select the **Use emails from report** check box.

File: docs/sources/enterprise/reporting.md [hunk 2]
```diff
@@ -42,6 +42,16 @@ Currently only Organization Admins can create reports.
...
+A test mail can be sent by pressing the Send test mail button. This will bring up a modal window where an email address or
+a list of email addresses can be entered. If the mail should use the mail addresses from the report instead,
+check the Use emails from report checkbox. 
+
+*Note that it is the last saved version of the report that will be sent and changes made to the report after saving will not be
```
Comment (oddlittlebird) [inline docs/sources/enterprise/reporting.md:50]: Rewrite this to be the intro to the numbered steps above, placed under the heading. 

The last saved version of the report will be sent to selected emails. You can use this to verify emails are working and to make sure the report generates and is displayed as you expect.

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
I have some suggestions.

File: docs/sources/enterprise/reporting.md (hunk 1 above)
Comment (jongyllen) [reply docs/sources/enterprise/reporting.md:45]: Thanks!

**Review by oddlittlebird — APPROVED** [review]

**Pull request merged**